### PR TITLE
Shaping codeanalyzer to work with DGI c2g

### PR DESCRIPTION
**Pull request by rahlk**
List of changes:

- Rename project name and cli: `code2graph` --to--> `codeanalyzer`
- Remove demos and other jars from the project. These will be hosted elsewhere. 
- Codeanalyzer now produces a single JSON consisting of a (context insensitive) SDG graph.
- The application has been dockerized. Once we create a container in quay.io, we may host it there.

### CODE REVIEW

**Review by rofrano — APPROVED** [review]
Looks good to me.

File: src/main/java/org/konveyor/dgi/GraphAugmentor.java
Comment (rofrano) [inline src/main/java/org/konveyor/dgi/GraphAugmentor.java:1]: Please add IBM Copyright to this Java file

File: src/main/java/org/konveyor/dgi/NodeFinder.java
Comment (rofrano) [inline src/main/java/org/konveyor/dgi/NodeFinder.java:1]: Please add IBM Copyright to this Java file

File: src/main/java/org/konveyor/dgi/WalaSPTGraph.java
Comment (rofrano) [inline src/main/java/org/konveyor/dgi/WalaSPTGraph.java:1]: Please add IBM Copyright to this Java file

File: src/main/java/org/konveyor/dgi/utils/Graph2JSON.java
Comment (rofrano) [inline src/main/java/org/konveyor/dgi/utils/Graph2JSON.java:1]: Please add IBM Copyright to this Java file

File: src/main/java/org/konveyor/dgi/utils/graph/AbstractGraphEdge.java
Comment (rofrano) [inline src/main/java/org/konveyor/dgi/utils/graph/AbstractGraphEdge.java:1]: Please add IBM Copyright to this Java file

File: src/main/java/org/konveyor/dgi/utils/graph/AbstractGraphNode.java
Comment (rofrano) [inline src/main/java/org/konveyor/dgi/utils/graph/AbstractGraphNode.java:1]: Please add IBM Copyright to this Java file

**Review by rofrano — CHANGES_REQUESTED** [review]
Please add IBM Copyright to all of the Java files.

Also if I'm reading this diff correctly, it looks like you may have renamed files outside of `git`. This makes git think they were deleted and looses all git history for that file. In the future use:

```bash
git mv oldFilename.java newFilename.java
```

So that git knows it was renamed and not deleted and recreated. This preserves the file history.

**Review by rofrano — CHANGES_REQUESTED** [review]
Please add IBM Copyright to all of the Java files.

Also if I'm reading this diff correctly, it looks like you may have renamed files outside of `git`. This makes git think they were deleted and looses all git history for that file. In the future use:

```bash
git mv oldFilename.java newFilename.java
```

So that git knows it was renamed and not deleted and recreated. This preserves the file history.

**Comment by rahlk** [comment]
@rofrano 
- I have added license information to all the files that I created.
- I have dockerized the application. Once we create a container in quay.io, we may host it there.

**Review by rofrano — APPROVED** [review]
Looks great!

**Pull request merged**